relay-dynamic-config/src/global.rs

```diff
@@ -23,6 +24,15 @@ pub struct GlobalConfig {
     /// Quotas that apply to all projects.
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pub quotas: Vec<Quota>,
+    /// Configuration for global inbound filters.
+    ///
+    /// These filters are merged with generic filters in project configs before
+    /// applying.
+    #[serde(
+        deserialize_with = "default_on_error",
+        skip_serializing_if = "Option::is_none"
+    )]
+    pub filters: Option<GenericFiltersConfig>,
```

Comment:
Wasn't there an `ErrorBoundary` before?

Reply:
Yes, but it's no longer needed.

We only extract metrics in the following cases:
- In processing Relays. We don't want to limit that further.
- When events are going to be dropped due to dynamic sampling. Since there's a sampling decision outside generic filters, we don't want to limit this either.
In other cases, the events are forwarded upstream. This means that `Option` is good enough.

Reply:
Even if not needed, restored it in [d7790ea](https://github.com/getsentry/relay/pull/3161/commits/d7790eab2cb3ba91d9269a1bddb685314db7b632) to make the difference in the future.